### PR TITLE
Add tests for Add Todo command & it's related tests

### LINKED ISSUE #128: Add tests for Add Todo command & it's related tests
Add tests for Add Todo command & related tests

### PR DESCRIPTION
**Pull request by vevek**
Closes #128

### CODE REVIEW

**Comment by vevek** [comment]
Done up the tests for AddTodoCommand and it's related tests.

File: src/test/java/seedu/address/logic/parser/AddressBookParserTest.java [hunk 1]
```diff
@@ -75,6 +78,16 @@ public void parseCommand_addEto() throws Exception {
         assertEquals(new AddEventCommand(projectIndex, event), command);
     }
 
+    @Test
+    public void parseCommand_addTto() throws Exception {
+        Todo todo = new TodoBuilder().withDescription("CS2106 Tutorial").build();
+        Index projectIndex = Index.fromOneBased(1);
```
Comment (lirc572) [inline src/test/java/seedu/address/logic/parser/AddressBookParserTest.java:84]: Perhaps we can change all occurences of `Index.fromOneBased(1)` to `INDEX_FIRST` in this file?

**Review by lirc572 — APPROVED** [review]
LGTM!

File: src/test/java/seedu/address/logic/parser/AddressBookParserTest.java (hunk 1 above)
Comment (vevek) [reply src/test/java/seedu/address/logic/parser/AddressBookParserTest.java:84]: Will resolve this at a later date. Placed in issues. :)

**Pull request merged**